avm/ptn/ai-platform/baseline/main.bicep

```diff
@@ -0,0 +1,331 @@
+output storageAccountResourceId string = storageAccount.outputs.resourceId
+
+@description('The name of the storage account.')
+output storageAccountName string = storageAccount.outputs.name
+
+@description('The resource ID of the container registry.')
+output containerRegistryResourceId string = containerRegistry.outputs.resourceId
+
+@description('The name of the container registry.')
+output containerRegistryName string = containerRegistry.outputs.name
+
+@description('The resource ID of the workspace hub.')
+output workspaceHubResourceId string = workspaceHub.outputs.resourceId
+
+@description('The name of the workspace hub.')
+output workspaceHubName string = workspaceHub.outputs.name
+
+// ================ //
+// Definitions      //
+// ================ //
+
+@discriminator('type')
+type OutboundRuleType = FqdnOutboundRuleType | PrivateEndpointOutboundRule | ServiceTagOutboundRule
+
+type FqdnOutboundRuleType = {
+  @sys.description('Required. Type of a managed network Outbound Rule of the  workspace hub. Only supported when \'isolationMode\' is \'AllowOnlyApprovedOutbound\'.')
+  type: 'FQDN'
+
+  @sys.description('Required. Fully Qualified Domain Name to allow for outbound traffic.')
+  destination: string
+
+  @sys.description('')
+  category: 'Dependency' | 'Recommended' | 'Required' | 'UserDefined'?
+}
+
+type PrivateEndpointOutboundRule = {
+  @sys.description('Required. Type of a managed network Outbound Rule of the workspace hub.')
+  type: 'PrivateEndpoint'
+
+  @sys.description('Required. Service Tag destination for a Service Tag Outbound Rule for the managed network of the workspace hub.')
+  destination: {
+    @sys.description('Required. The resource ID of the target resource for the private endpoint.')
+    serviceResourceId: string
+
+    @sys.description('Optional. Whether the private endpoint can be used by jobs running on Spark.')
+    sparkEnabled: bool?
+
+    @sys.description('Required. The sub resource to connect for the private endpoint.')
+    subresourceTarget: string
+  }
+
+  @sys.description('')
+  category: 'Dependency' | 'Recommended' | 'Required' | 'UserDefined'?
+}
+
+type ServiceTagOutboundRule = {
+  @sys.description('Required. Type of a managed network Outbound Rule of the workspace hub. Only supported when \'isolationMode\' is \'AllowOnlyApprovedOutbound\'.')
+  type: 'ServiceTag'
+
+  @sys.description('Required. Service Tag destination for a Service Tag Outbound Rule for the managed network of the workspace hub.')
+  destination: {
+    @sys.description('Required. The name of the service tag to allow.')
+    portRanges: string
+
+    @sys.description('Required. The protocol to allow. Provide an asterisk(*) to allow any protocol.')
+    protocol: 'TCP' | 'UDP' | 'ICMP' | '*'
+
+    @sys.description('Required. Which ports will be allow traffic by this rule. Provide an asterisk(*) to allow any port.')
+    serviceTag: string
+  }
+
+  @sys.description('')
```

Comment:
Please note that a couple properties don't have a description. Please check.

Reply:
Sorry, copied from the other PR before it was updated 🙈

Reply:
Added